lib/widgets/message_list.dart

```diff
@@ -1262,15 +1270,24 @@ class MessageWithPossibleSender extends StatelessWidget {
           if (senderRow != null)
             Padding(padding: const EdgeInsets.fromLTRB(16, 2, 16, 0),
               child: senderRow),
-          EditStateMarker(
-            editState: message.editState,
+          Row(
+            crossAxisAlignment: CrossAxisAlignment.baseline,
+            textBaseline: localizedTextBaseline(context),
             children: [
+              const SizedBox(width: 16),
               Expanded(child: Column(
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
                   MessageContent(message: message, content: item.content),
                   if ((message.reactions?.total ?? 0) > 0)
-                    ReactionChipsList(messageId: message.id, reactions: message.reactions!)
+                    ReactionChipsList(messageId: message.id, reactions: message.reactions!),
+                  if (editStateText != null)
+                    Text(editStateText, textAlign: TextAlign.end,
+                      style: TextStyle(
+                        color: messageListTheme.editStateLabel,
+                        fontFamily: 'Source Sans 3',
+                        fontSize: 12,
+                        height: (12 / 12))),
```

Comment:
I guess the Figma isn't clear whether the reaction chips or the marker should come first, vertically. How about putting the marker first? In the case where there are dozens of reaction chips that wrap onto many lines, I think it would be neater and clearer if the EDITED or MOVED marker were above them instead of below them.

Reply:
Yeah that sounds good to me.

Reply:
Just to be sure, started a [disccusion on CZO](https://chat.zulip.org/#narrow/stream/48-mobile/topic/check-x.20icons.20in.20left.20margin/near/1931552).